src/components/Book.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+// const ff = () => fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/a0EBuQKzzPLsbomuWCDX/books',
+//   {
+//     method: 'POST',
+//     headers: {
+//       'Content-type': 'application/json; charset=UTF-8',
+//     },
+//     body: JSON.stringify({
+//       item_id: "item1",
+//       title: "The Great Gatsby",
+//       author: "John Smith",
+//       category: "Fiction"
+//     }),
+//   }).then((resp) => resp.text());
+//   console.log(ff());
+
```

Comment:
- [ ] It will be nice to remove commented code from you code base as this makes your code dirty and less readable.